app/src/main/java/com/onelist/RetrofitClientInstance.kt

```diff
@@ -12,7 +12,7 @@ object RetrofitClientInstance {
             // has this object been created yet?
             if (retrofit == null) {
                 // create it
-                retrofit = retrofit2.Retrofit.Builder()
+                retrofit = Retrofit.Builder()
```

Comment:
its better to omit redundant words like retrofit2 as it makes the code more readible and it does not add value to the code as it is not nessassary to state it explicitily